### PR TITLE
fix: fix execute_deposit_utils

### PR DESCRIPTION
**Pull request by sparqet**
# Pull Request type

Please add the labels corresponding to the type of changes your PR introduces:

- Bugfix

## What is the new behavior?

- fix execute_deposit_utils

## Does this introduce a breaking change?

No

### CODE REVIEW

File: src/deposit/execute_deposit_utils.cairo [hunk 1]
```diff
@@ -416,38 +413,38 @@ fn execute_deposit_helper(
...
+            fees.amount_after_fees * _params.token_in_price.min, pool_value, market_tokens_supply,
         );
 
     market_utils::apply_delta_to_pool_amount(
         *params.data_store,
         *params.event_emitter,
         _params.market,
-        _params.token_out,
-        to_signed(amount_after_fees + fees.fee_amount_for_pool, true),
+        _params.token_in,
```
Comment (Sk8erboi84) [inline src/deposit/execute_deposit_utils.cairo:446]: its token_out

**Review by Sk8erboi84 — CHANGES_REQUESTED** [review]

File: src/deposit/execute_deposit_utils.cairo [hunk 2]
```diff
@@ -162,10 +162,9 @@ fn execute_deposit(params: ExecuteDepositParams) {
                 deposit.ui_fee_receiver,
             );
 
-    assert(
-        cache.long_token_amount == 0 && cache.short_token_amount == 0,
-        DepositError::EMPTY_DEPOSIT_AMOUNTS_AFTER_SWAP
-    );
+    if cache.long_token_amount == 0 && cache.short_token_amount == 0 {
```
Comment (Sk8erboi84) [inline src/deposit/execute_deposit_utils.cairo:165]: why did you remove the assert?

**Review by Sk8erboi84 — CHANGES_REQUESTED** [review]

File: src/deposit/execute_deposit_utils.cairo (hunk 2 above)
Comment (sparqet) [reply src/deposit/execute_deposit_utils.cairo:165]: Because it has to be and if not an assert

File: src/deposit/execute_deposit_utils.cairo (hunk 1 above)
Comment (sparqet) [reply src/deposit/execute_deposit_utils.cairo:446]: No it's token_in

**Review by Sk8erboi84 — APPROVED** [review]
